### PR TITLE
Store: Update all instances of title case to sentence case

### PR DESCRIPTION
**Pull request by kyleaparker**
#### Changes proposed in this Pull Request

* Store is currently using majority sentence case with a few instances of title case in buttons, labels, and headings
* Updates all instances to use sentence case

**Before Example**
Before-Promotions

**After Example**

After-Promotions

#### Testing instructions

This PR touches most screens in Store
*  Navigate to http://calypso.localhost:3000/store/
*  Navigate through each screen in store checking that all buttons, titles, and labels are using sentence case

Fixes https://github.com/Automattic/wp-calypso/issues/33438

### CODE REVIEW

File: client/extensions/woocommerce/app/order/create.js [hunk 1]
```diff
@@ -143,7 +143,7 @@ class Order extends Component {
 						busy={ isSaving }
 						disabled={ ! hasOrderEdits || isSaving }
 					>
-						{ translate( 'Save Order' ) }
+						{ translate( 'Save order' ) }
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/order/create.js]: :ok: This change is safe, we'll keep the existing 13 translations.

File: client/extensions/woocommerce/app/order/header.js [hunk 2]
```diff
@@ -132,7 +132,7 @@ class OrderActionHeader extends Component {
 
 		const buttons = [
 			<Button key="edit" primary onClick={ this.toggleEditing }>
-				{ translate( 'Edit Order' ) }
+				{ translate( 'Edit order' ) }
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/order/header.js]: :ok: This change is safe, we'll keep the existing 1 translations.

File: client/extensions/woocommerce/app/order/order-details/index.js [hunk 3]
```diff
@@ -95,7 +95,7 @@ class OrderDetails extends Component {
 		return (
 			<div className="order-details">
 				<SectionHeader
-					label={ translate( 'Order %(orderId)s Details', {
+					label={ translate( 'Order %(orderId)s details', {
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/order/order-details/index.js]: :disappointed: 1 existing translations will be lost with this change.

File: client/extensions/woocommerce/app/order/order-activity-log/index.js [hunk 4]
```diff
@@ -25,7 +25,7 @@ class OrderActivityLog extends Component {
 
 		return (
 			<div className="order-activity-log">
-				<SectionHeader label={ translate( 'Activity Log' ) } />
+				<SectionHeader label={ translate( 'Activity log' ) } />
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/order/order-activity-log/index.js]: :ok: This change is safe, we'll keep the existing 20 translations.

File: client/extensions/woocommerce/app/order/order-activity-log/new-note.js [hunk 5]
```diff
@@ -74,11 +74,11 @@ class CreateOrderNote extends Component {
 				</FormFieldSet>
 				<div className="order-activity-log__new-note-type">
 					<FormSelect onChange={ this.setType } value={ this.state.type }>
-						<option value={ 'internal' }>{ translate( 'Private Note' ) }</option>
-						<option value={ 'email' }>{ translate( 'Send to Customer' ) }</option>
+						<option value={ 'internal' }>{ translate( 'Private note' ) }</option>
+						<option value={ 'email' }>{ translate( 'Send to customer' ) }</option>
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/order/order-activity-log/new-note.js]: :ok: This change is safe, we'll keep the existing 1 translations.

File: client/extensions/woocommerce/app/order/order-activity-log/new-note.js [hunk 6]
```diff
@@ -74,11 +74,11 @@ class CreateOrderNote extends Component {
...
 					</FormSelect>
 					<Button primary onClick={ this.saveNote } busy={ isNoteSaving } disabled={ isNoteSaving }>
-						{ translate( 'Add Note' ) }
+						{ translate( 'Add note' ) }
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/order/order-activity-log/new-note.js]: :ok: This change is safe, we'll keep the existing 46 translations.

File: client/extensions/woocommerce/app/order/order-customer/index.js [hunk 7]
```diff
@@ -130,12 +130,12 @@ class OrderCustomerInfo extends Component {
 
 		return (
 			<div className="order-customer">
-				<SectionHeader label={ translate( 'Customer Information' ) } />
+				<SectionHeader label={ translate( 'Customer information' ) } />
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/order/order-customer/index.js]: :disappointed: 1 existing translations will be lost with this change.

File: client/extensions/woocommerce/app/order/order-customer/index.js [hunk 8]
```diff
@@ -130,12 +130,12 @@ class OrderCustomerInfo extends Component {
...
 				<Card>
 					<div className="order-customer__container">
 						<div className="order-customer__billing">
 							<h3 className="order-customer__billing-details">
-								{ translate( 'Billing Details' ) }
+								{ translate( 'Billing details' ) }
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/order/order-customer/index.js]: :disappointed: 38 existing translations will be lost with this change.

File: client/extensions/woocommerce/app/order/order-customer/index.js [hunk 9]
```diff
@@ -162,7 +162,7 @@ class OrderCustomerInfo extends Component {
 
 						<div className="order-customer__shipping">
 							<h3 className="order-customer__shipping-details">
-								{ translate( 'Shipping Details' ) }
+								{ translate( 'Shipping details' ) }
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/order/order-customer/index.js]: :disappointed: 3 existing translations will be lost with this change.

File: client/extensions/woocommerce/app/order/order-details/fee-dialog.js [hunk 10]
```diff
@@ -107,7 +107,7 @@ class OrderFeeDialog extends Component {
 		const dialogButtons = [
 			<Button onClick={ closeDialog }>{ translate( 'Cancel' ) }</Button>,
 			<Button primary onClick={ this.handleFeeSave } disabled={ ! canSave }>
-				{ translate( 'Add Fee' ) }
+				{ translate( 'Add fee' ) }
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/order/order-details/fee-dialog.js]: :disappointed: 1 existing translations will be lost with this change.
If the string must change, the following option has already been translated **1** time:
`translate( 'Add a fee' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/extensions/woocommerce/app/order/order-payment/index.js [hunk 11]
```diff
@@ -100,9 +100,9 @@ class OrderPaymentCard extends Component {
 		if ( 'refunded' === order.status || codProcessing ) {
 			return null;
 		} else if ( isOrderWaitingPayment( order.status ) ) {
-			return <Button onClick={ this.markAsPaid }>{ translate( 'Mark as Paid' ) }</Button>;
+			return <Button onClick={ this.markAsPaid }>{ translate( 'Mark as paid' ) }</Button>;
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/order/order-payment/index.js]: :ok: This change is safe, we'll keep the existing 1 translations.

File: client/extensions/woocommerce/app/order/order-payment/index.js [hunk 12]
```diff
@@ -100,9 +100,9 @@ class OrderPaymentCard extends Component {
...
 		}
-		return <Button onClick={ this.toggleDialog }>{ translate( 'Submit Refund' ) }</Button>;
+		return <Button onClick={ this.toggleDialog }>{ translate( 'Submit refund' ) }</Button>;
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/order/order-payment/index.js]: :ok: This change is safe, we'll keep the existing 1 translations.

File: client/extensions/woocommerce/app/orders/orders-filter-nav.js [hunk 13]
```diff
@@ -31,11 +31,11 @@ class OrdersFilterNav extends Component {
 
 	render() {
 		const { translate, site, status } = this.props;
-		let currentSelection = translate( 'All Orders' );
+		let currentSelection = translate( 'All orders' );
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/orders/orders-filter-nav.js]: :disappointed: 1 existing translations will be lost with this change.

File: client/extensions/woocommerce/app/orders/orders-filter-nav.js [hunk 14]
```diff
@@ -31,11 +31,11 @@ class OrdersFilterNav extends Component {
...
 		if ( ORDER_UNPAID === status ) {
-			currentSelection = translate( 'Awaiting Payment' );
+			currentSelection = translate( 'Awaiting payment' );
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/orders/orders-filter-nav.js]: :disappointed: 1 existing translations will be lost with this change.

File: client/extensions/woocommerce/app/orders/orders-filter-nav.js [hunk 15]
```diff
@@ -31,11 +31,11 @@ class OrdersFilterNav extends Component {
...
 		} else if ( ORDER_UNFULFILLED === status ) {
-			currentSelection = translate( 'Awaiting Fulfillment' );
+			currentSelection = translate( 'Awaiting fulfillment' );
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/orders/orders-filter-nav.js]: :disappointed: 1 existing translations will be lost with this change.

File: client/extensions/woocommerce/app/promotions/promotion-header.js [hunk 16]
```diff
@@ -45,13 +45,13 @@ function renderSaveButton( onSave, isBusy, label ) {
 const PromotionHeader = ( { promotion, onSave, onTrash, isBusy, translate, site } ) => {
 	const existing = promotion && ! isObject( promotion.id );
 	const trashButton = renderTrashButton( onTrash, isBusy, translate( 'Delete' ) );
-	const saveLabel = existing ? translate( 'Update' ) : translate( 'Save & Publish' );
+	const saveLabel = existing ? translate( 'Update' ) : translate( 'Save & publish' );
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/promotions/promotion-header.js]: :disappointed: 1 existing translations will be lost with this change.

File: client/extensions/woocommerce/index.js [hunk 17]
```diff
@@ -62,31 +62,31 @@ const getStorePages = () => {
...
 			configKey: 'woocommerce/extension-products',
-			documentTitle: translate( 'New Product' ),
+			documentTitle: translate( 'New product' ),
 			path: '/store/product/:site',
 		},
 		{
 			container: ProductUpdate,
 			configKey: 'woocommerce/extension-products',
-			documentTitle: translate( 'Edit Product' ),
+			documentTitle: translate( 'Edit product' ),
```
Comment (a8ci18n) [inline client/extensions/woocommerce/index.js]: :disappointed: 1 existing translations will be lost with this change.

File: client/extensions/woocommerce/index.js [hunk 18]
```diff
@@ -122,13 +122,13 @@ const getStorePages = () => {
...
 			configKey: 'woocommerce/extension-promotions',
-			documentTitle: translate( 'New Promotion' ),
+			documentTitle: translate( 'New promotion' ),
 			path: '/store/promotion/:site',
 		},
 		{
 			container: PromotionUpdate,
 			configKey: 'woocommerce/extension-promotions',
-			documentTitle: translate( 'Edit Promotion' ),
+			documentTitle: translate( 'Edit promotion' ),
```
Comment (a8ci18n) [inline client/extensions/woocommerce/index.js]: :disappointed: 1 existing translations will be lost with this change.

File: client/extensions/woocommerce/app/promotions/promotion-models.js [hunk 19]
```diff
@@ -67,9 +67,9 @@ const appliesToCoupon = {
 			component: (
 				<PromotionAppliesToField
 					selectionTypes={ [
-						{ labelText: translate( 'All Products' ), type: 'all' },
-						{ labelText: translate( 'Specific Products' ), type: 'productIds' },
-						{ labelText: translate( 'Product Categories' ), type: 'productCategoryIds' },
+						{ labelText: translate( 'All products' ), type: 'all' },
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/promotions/promotion-models.js]: :ok: This change is safe, we'll keep the existing 55 translations.

File: client/extensions/woocommerce/app/promotions/promotion-models.js [hunk 20]
```diff
@@ -67,9 +67,9 @@ const appliesToCoupon = {
...
+						{ labelText: translate( 'Specific products' ), type: 'productIds' },
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/promotions/promotion-models.js]: :ok: This change is safe, we'll keep the existing 1 translations.

File: client/extensions/woocommerce/index.js [hunk 21]
```diff
@@ -62,31 +62,31 @@ const getStorePages = () => {
...
 			configKey: 'woocommerce/extension-products',
-			documentTitle: translate( 'Edit Product' ),
+			documentTitle: translate( 'Edit product' ),
 			path: '/store/product/:site/:product_id',
 		},
 		{
 			container: ProductCategories,
 			configKey: 'woocommerce/extension-product-categories',
-			documentTitle: translate( 'Product Categories' ),
+			documentTitle: translate( 'Product categories' ),
```
Comment (a8ci18n) [inline client/extensions/woocommerce/index.js]: :disappointed: 52 existing translations will be lost with this change.

File: client/extensions/woocommerce/app/promotions/promotion-models.js [hunk 22]
```diff
@@ -155,7 +155,7 @@ const EndDateField = props => {
 
 	return (
 		<DateField
-			labelText={ translate( 'End Date' ) }
+			labelText={ translate( 'End date' ) }
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/promotions/promotion-models.js]: :ok: This change is safe, we'll keep the existing 79 translations.

File: client/extensions/woocommerce/app/promotions/promotion-models.js [hunk 23]
```diff
@@ -169,11 +169,11 @@ const EndDateField = props => {
 const productSaleModel = {
 	appliesToProductSale,
 	productAndSalePrice: {
-		labelText: translate( 'Product & Sale Price' ),
+		labelText: translate( 'Product & sale price' ),
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/promotions/promotion-models.js]: :ok: This change is safe, we'll keep the existing 1 translations.

File: client/extensions/woocommerce/app/promotions/promotion-models.js [hunk 24]
```diff
@@ -169,11 +169,11 @@ const EndDateField = props => {
...
 		cssClass: 'promotions__promotion-form-card-primary',
 		fields: {
 			salePrice: {
-				component: <CurrencyField labelText={ translate( 'Product Sale Price' ) } isRequired />,
+				component: <CurrencyField labelText={ translate( 'Product sale price' ) } isRequired />,
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/promotions/promotion-models.js]: Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Product & Sale Price' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update-sentence-case)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/extensions/woocommerce/app/promotions/promotion-models.js [hunk 25]
```diff
@@ -263,7 +263,7 @@ const couponConditions = {
 const fixedProductModel = {
 	appliesToCoupon,
 	couponCodeAndDiscount: {
-		labelText: translate( 'Coupon Code & Discount' ),
+		labelText: translate( 'Coupon code & discount' ),
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/promotions/promotion-models.js]: :disappointed: 1 existing translations will be lost with this change.

File: client/extensions/woocommerce/app/promotions/promotion-models.js [hunk 26]
```diff
@@ -273,7 +273,7 @@ const fixedProductModel = {
 			fixedDiscount: {
 				component: (
 					<CurrencyField
-						labelText={ translate( 'Product Discount', { comment: 'for coupon' } ) }
+						labelText={ translate( 'Product discount', { comment: 'for coupon' } ) }
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/promotions/promotion-models.js]: :ok: This change is safe, we'll keep the existing 1 translations.

File: client/extensions/woocommerce/app/promotions/promotion-models.js [hunk 27]
```diff
@@ -303,7 +303,7 @@ const fixedCartModel = {
 			fixedDiscount: {
 				component: (
 					<CurrencyField
-						labelText={ translate( 'Cart Discount', { comment: 'for coupon' } ) }
+						labelText={ translate( 'Cart discount', { comment: 'for coupon' } ) }
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/promotions/promotion-models.js]: :ok: This change is safe, we'll keep the existing 1 translations.

File: client/extensions/woocommerce/app/promotions/promotion-models.js [hunk 28]
```diff
@@ -333,7 +333,7 @@ const percentCartModel = {
 			percentDiscount: {
 				component: (
 					<PercentField
-						labelText={ translate( 'Percent Cart Discount', { comment: 'for coupon' } ) }
+						labelText={ translate( 'Percent cart discount', { comment: 'for coupon' } ) }
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/promotions/promotion-models.js]: :ok: This change is safe, we'll keep the existing 1 translations.

File: client/extensions/woocommerce/app/settings/payments/payment-method-bacs.js [hunk 29]
```diff
@@ -127,13 +127,13 @@ class PaymentMethodBACS extends Component {
 					/>
 				</FormFieldset>
 				<FormFieldset className="payments__method-edit-field-container">
-					<FormLabel>{ translate( 'Account Details' ) }</FormLabel>
+					<FormLabel>{ translate( 'Account details' ) }</FormLabel>
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/settings/payments/payment-method-bacs.js]: :ok: This change is safe, we'll keep the existing 53 translations.

File: client/extensions/woocommerce/app/settings/payments/payments-location-currency.js [hunk 30]
```diff
@@ -89,7 +89,7 @@ class SettingsPaymentsLocationCurrency extends Component {
 				<Card className="payments__address-currency-container">
 					<StoreAddress showLabel={ false } />
 					<div className="payments__currency-container">
-						<FormLabel>{ translate( 'Store Currency' ) }</FormLabel>
+						<FormLabel>{ translate( 'Store currency' ) }</FormLabel>
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/settings/payments/payments-location-currency.js]: :ok: This change is safe, we'll keep the existing 1 translations.

File: client/extensions/woocommerce/app/settings/shipping/shipping-origin.js [hunk 31]
```diff
@@ -19,7 +19,7 @@ const ShippingOrigin = ( { translate, onChange } ) => {
 	return (
 		<div className="shipping__origin">
 			<ExtendedHeader
-				label={ translate( 'Shipping Origin' ) }
+				label={ translate( 'Shipping origin' ) }
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/settings/shipping/shipping-origin.js]: :ok: This change is safe, we'll keep the existing 1 translations.

File: client/extensions/woocommerce/app/settings/shipping/shipping-zone-list.js [hunk 32]
```diff
@@ -102,7 +102,7 @@ class ShippingZoneList extends Component {
 				<QueryShippingZones siteId={ siteId } />
 				<QuerySettingsGeneral siteId={ siteId } />
 				<ExtendedHeader
-					label={ translate( 'Shipping Zones' ) }
+					label={ translate( 'Shipping zones' ) }
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/settings/shipping/shipping-zone-list.js]: :ok: This change is safe, we'll keep the existing 5 translations.

File: client/extensions/woocommerce/app/settings/shipping/shipping-zone/shipping-zone-header.js [hunk 33]
```diff
@@ -32,9 +32,9 @@ const ShippingZoneHeader = ( {
 } ) => {
 	const currentCrumb =
 		zone && isNumber( zone.id ) ? (
-			<span>{ translate( 'Edit Shipping Zone' ) }</span>
+			<span>{ translate( 'Edit shipping zone' ) }</span>
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/settings/shipping/shipping-zone/shipping-zone-header.js]: :ok: This change is safe, we'll keep the existing 5 translations.

File: client/extensions/woocommerce/app/settings/shipping/shipping-zone/shipping-zone-header.js [hunk 34]
```diff
@@ -32,9 +32,9 @@ const ShippingZoneHeader = ( {
...
 		) : (
-			<span>{ translate( 'Add new Shipping Zone' ) }</span>
+			<span>{ translate( 'Add new shipping zone' ) }</span>
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/settings/shipping/shipping-zone/shipping-zone-header.js]: :ok: This change is safe, we'll keep the existing 1 translations.

File: client/extensions/woocommerce/app/settings/shipping/shipping-zone/shipping-zone-location-dialog-settings.js [hunk 35]
```diff
@@ -182,7 +182,7 @@ const ShippingZoneLocationDialogSettings = ( {
 	return (
 		<div className="shipping-zone__location-dialog-settings">
 			<FormFieldSet>
-				<FormLegend>{ translate( 'Shipping Zone settings' ) }</FormLegend>
+				<FormLegend>{ translate( 'Shipping zone settings' ) }</FormLegend>
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/settings/shipping/shipping-zone/shipping-zone-location-dialog-settings.js]: :ok: This change is safe, we'll keep the existing 1 translations.

File: client/extensions/woocommerce/app/settings/taxes/taxes-wcs.js [hunk 36]
```diff
@@ -144,7 +144,7 @@ class SettingsTaxesWooCommerceServices extends Component {
 		return (
 			<div className="taxes__nexus">
 				<ExtendedHeader
-					label={ translate( 'Store Address' ) }
+					label={ translate( 'Store address' ) }
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/settings/taxes/taxes-wcs.js]: :ok: This change is safe, we'll keep the existing 38 translations.

File: client/extensions/woocommerce/components/address-view/index.js [hunk 37]
```diff
@@ -129,7 +129,7 @@ class AddressView extends Component {
 				onClick={ this.onClickShowAddressLine2 }
 			>
 				<Gridicon icon="plus-small" />
-				{ translate( 'Add Address Line 2' ) }
+				{ translate( 'Add address line 2' ) }
```
Comment (a8ci18n) [inline client/extensions/woocommerce/components/address-view/index.js]: :ok: This change is safe, we'll keep the existing 1 translations.

File: client/extensions/woocommerce/index.js [hunk 38]
```diff
@@ -62,31 +62,31 @@ const getStorePages = () => {
 		{
 			container: ProductCreate,
 			configKey: 'woocommerce/extension-products',
-			documentTitle: translate( 'New Product' ),
+			documentTitle: translate( 'New product' ),
```
Comment (a8ci18n) [inline client/extensions/woocommerce/index.js]: :ok: This change is safe, we'll keep the existing 2 translations.

File: client/extensions/woocommerce/index.js [hunk 39]
```diff
@@ -104,13 +104,13 @@ const getStorePages = () => {
 		{
 			container: Order,
 			configKey: 'woocommerce/extension-orders',
-			documentTitle: translate( 'Order Details' ),
+			documentTitle: translate( 'Order details' ),
```
Comment (a8ci18n) [inline client/extensions/woocommerce/index.js]: :disappointed: 46 existing translations will be lost with this change.

File: client/extensions/woocommerce/index.js [hunk 40]
```diff
@@ -122,13 +122,13 @@ const getStorePages = () => {
 		{
 			container: PromotionCreate,
 			configKey: 'woocommerce/extension-promotions',
-			documentTitle: translate( 'New Promotion' ),
+			documentTitle: translate( 'New promotion' ),
```
Comment (a8ci18n) [inline client/extensions/woocommerce/index.js]: :ok: This change is safe, we'll keep the existing 1 translations.

**Comment by kyleaparker** [comment]
>Not saying we should definitely do this, but we might consider applying text-transform: capitalize; to the Button component(s) stylesheet. 

This wouldn't work in this case since we only want the first word capitalized, I don't think there is an option for that. I'm not exactly sure how to go about fixing the translation errors on this though.

**Comment by jameskoster** [comment]
Whoops, I was looking at the diffs the wrong way around 🤦‍♂ 

In that case we could use the `:first-letter` selector [like so](https://codepen.io/jameskoster/pen/rEMgmZ). It requires the "parent" to be a block element, but since the `.button` class in Calypso has `display: inline-block;` I don't think that would be a problem.

If we wanted to get fancy we could even create utility classes, or a mixin for different cases and update the `Button` component to include that.

**Comment by kyleaparker** [comment]
>In that case we could use the :first-letter selector like so. It requires the "parent" to be a block element, but since the .button class in Calypso has display: inline-block; I don't think that would be a problem.

This seems like a good approach, only problem is when buttons or heading include proper nouns, trademarks etc like "Setup your Stripe account" where Stripe still needs to be capitalized

**Comment by sixhours** [comment]
Flagged this for i18n review to see how they would prefer to handle situations like this, where many strings would need to be re-translated.

**Comment by sixhours** [comment]
Thanks @akirk! With that in mind, this looks good to me @kyleaparker!

**Pull request merged**